test/unit/spec/Util_spec.js

```diff
@@ -289,5 +292,100 @@ describe('util/Util', () => {
       expect(Logger.error).toHaveBeenCalledTimes(1);
       expect(Logger.error).toHaveBeenCalledWith('Cannot redirect to empty URL: ()');
     });
+
+  });
+
+  describe('enrollmentRedirect', () => {
+
+    class EnrollmentView {
+      add() {}
+      options = { appState: { getCurrentViewState() {} }}
+    }
+
+    const enrollmentView = new EnrollmentView();
+
+    beforeEach(() => {
+      jest.spyOn(utilSpy, 'redirectWithFormGet').mockReturnValue(() => {});
+      jest.spyOn(enrollmentView.options.appState, 'getCurrentViewState').mockReturnValue({
+        href:'https://org.okta.com/login/token/redirect?stateToken=mockedStateToken123'});
+
+    });
+
+    afterEach(() => {
+      delete window.location;
+    });
+
+    it('add Open OV button', () => {
+      // create mocks
+      let expectedAddArgs = [];
+      jest.spyOn(BrowserFeatures, 'isAndroid').mockReturnValue(true);
+      spyOn(enrollmentView, 'add').and.callFake((addArg) => {expectedAddArgs.push(addArg);});
+
+      Object.defineProperty(window, 'location', {
+        value: {
+          href: 'https://garbage%redirect_uri=https%3A%2F%2Flogin.okta.com%garbage',
```

Comment:
I don't _think_ it's important, but the encoding looks wrong for the href. `%` alone isn't valid.